pinot-connectors/pinot-flink-connector/pom.xml

```diff
@@ -96,11 +96,14 @@
           See also: properties section and profiles section.
         -->
         <configuration>
-          <testFailureIgnore>false</testFailureIgnore>
           <!-- Excludes integration tests when unit tests are run. -->
           <excludes>
             <exclude>**/*ITCase.java</exclude>
           </excludes>
+          <systemPropertyVariables>
+            <!-- org.glassfish.grizzly uses JUL, -->
+            <java.util.logging.config.file>src/test/resources/logging-test.properties</java.util.logging.config.file>
+          </systemPropertyVariables>
```

Comment:
Couldn't we just set this in the root pom.xml?

Reply:
I don't know what the Java Util Logging framework will do if this file is not available. This is the fast and simple solution. It  might be worth checking if this AND jul-to-slf4j can be replaced everywhere with slf4j-jdk-platform-logging. jul-to-slf4 is a hack with possible huge performance penalty, whereas slf4j-jdk-platform-logging is the proper way to log to slf4j via a logging provider